src/main/java/com/akademia/educators_back/controller/ProblemController.java

```diff
@@ -0,0 +1,37 @@
+package com.akademia.educators_back.controller;
+
+import com.akademia.educators_back.service.impl.ProblemService;
+import com.akademia.educators_back.to.ProblemTo;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@RequestMapping("api/problems")
+@RestController
+@RequiredArgsConstructor
+public class ProblemController {
+
+    public ProblemService problemService;
+
+    @GetMapping("/api/problemsList")
```

Comment:
czyli nasza ścieżka to "api/problems/api/problemList" -> "api/problems/list" lub "api/problems"